### DISCUSSION TITLE
Misleading documentation

### DISCUSSION THREAD

**Discussion by OsmanM94** [supabase/supabase#29150] [Questions] [answered]
This is a copy/paste from the documentation. The docs state that this is an asynchronous operation however, it is not. This is misleading, and there is also a massive typo in the "Subscribe / Listen to a specific table" section of the official documentation.

However, I'm more interested in the asynchronous operation. Nearly all of them, from subscribing to a channel, are not supported asynchronously, even though the documentation claims they are. Please address this issue.

https://supabase.com/docs/reference/swift/subscribe

<img width="1160" alt="supabase listener" src="https://github.com/user-attachments/assets/68c5323a-6d59-45f2-94c8-2f1923119ab9">

**Comment by grdsdev** [comment, marked as answer]
Hi @OsmanM94 realtime used to need await, as it was implemented as an actor before, now that isn't needed anymore, you can drop the await from most of the api calls.

PR updating docs: https://github.com/supabase/supabase/pull/29183

Let me know if you have any other topic realtime related.
